Tests/Indicators/ArnaudLegouxMovingAverageTest.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
```

Comment:
Sorry, no need for the XML comments in the test project (the XML documentation flag is disabled in the Tests project)